test/kwtool/accounts_test.exs

```diff
@@ -0,0 +1,99 @@
+defmodule Kwtool.AccountsTest do
+  use Kwtool.DataCase
+
+  alias Kwtool.Accounts
+  alias Kwtool.Accounts.User
+
+  describe "changeset/2" do
```

Comment:
These tests need to be reformatted. We pass the function name and rarity in `describe/2` and message in `test/2`.
For the detail, you can [check here](https://nimblehq.co/compass/development/code-conventions/ex-unit/#formatting)
```elixir
describe "list_users/0" do
  test "returns all users" do
      user = insert(:user)
      assert Accounts.list_users() == [user]
    end
end
```

Reply:
updated.